src/config/backpack/crud.php

```diff
@@ -13,6 +13,7 @@
     */
 
     'show_translatable_field_icon' => true,
+    'override_available_locales_on_create' => true,
```

Comment:
This can't be true by default or it would be a BC. 
```suggestion
    'override_available_locales_on_create' => false,
```